tests/run-tests.ps1

```diff
@@ -9,7 +9,7 @@ param(
     [string]$OSFilter,
     [string]$Registry,
     [string]$RepoPrefix,
-    [string]$TestCategory = "all",
+    [array]$TestCategories,
```

Comment:
Shouldn't this be identical to the build-and-test script parameter? It should have the ValidateSet attribute.